src/benchmark.py

```diff
@@ -0,0 +1,235 @@
+    # Prepare schema for table
+    df[:0].to_sql('temp', engine, if_exists = "replace", index=False)
+    df.columns = cleanColumns(df.columns)
+    # replace mode
+    if mode == 'replace':
+        cur.execute("DROP TABLE " + table_name)
+        cur.connection.commit()
+    # Prepare data to temp
+    if psql == 'psql':
+        data = StringIO.StringIO()
+        df.to_csv(data, header=False, index=False)
+        data.seek(0)
+        sql_query = """
+            COPY temp FROM STDIN WITH
+                CSV
+                HEADER;
+        """
+        cur.copy_expert(sql=sql_query, file=data)
+        cur.connection.commit()
+        # Copy or append table temp to target table
+        if (mode == 'replace' or new_table == 0):
+            sql_query2 = """
+                ALTER TABLE temp
+                RENAME TO %s;
+            """
+        else:
+            sql_query2 = """
+                INSERT INTO %s SELECT * FROM temp;
+                DROP TABLE temp;
+            """
+        cur.execute(sql_query2 % table_name)
+        cur.connection.commit()
+    else:
+        with s3.open('rxminer/temp.csv', 'wb') as f:
+            df.to_csv(f, index=False, header=False)
+        sql_query = """
+            COPY temp
+            FROM 's3://rxminer/temp.csv'
+            IAM_ROLE 'arn:aws:iam::809946175142:role/RedshiftCopyUnload'
+            CSV
+            IGNOREHEADER 1;
+            COMMIT;
+        """
+        cur.execute(sql_query)
+        # Copy or append table temp to target table
+        if (mode == 'replace' or new_table == 0):
+            sql_query2 = """
+                ALTER TABLE temp
+                RENAME TO %s;
+                COMMIT;VACUUM;COMMIT;
+            """
+        else:
+            sql_query2 = """
+                INSERT INTO %s SELECT * FROM temp;
+                DROP TABLE temp;
+                COMMIT;VACUUM;COMMIT;
+            """
+        cur.execute(sql_query2 % table_name)
+
+def read_medicaid(year, mode):
+    type_dir = 'sdud/medicaid_sdud_'
+    table_name = 'sdudtest'
+    cols_to_keep = [1,5,7,9,10,11,12,13,19]
+    column_names = [
+        'state', 'year', 'drug_name',
+        'unit_reimbursed', 'num_prescriptions',
+        'tot_reimbursed', 'medicaid_reimbursed',
+        'nonmedicaid_reimbursed', 'ndc'
+        ]
+    d_type = {'ndc': str}
+    chunks = pd.read_csv(
+        s3_path+type_dir+str(year)+'.csv',
+        usecols = cols_to_keep,
+        names = column_names,
+        dtype = d_type,
+        skiprows = 1,
+        chunksize = chunk_size)
+    new_table = 0
+    for chunk in chunks:
+        chunk.dropna(subset=['tot_reimbursed'], inplace=True)
+        if pd2db == "yes":
+            pd_to_postgres(chunk, table_name, mode)
+        else:
+            df_to_sql(chunk, table_name, mode, new_table, psql, cur, engine)
+        new_table += 1
+        print(datetime.datetime.now(eastern).strftime("%Y-%m-%dT%H:%M:%S.%f")+' Medicaid data: reading in progress...')
```

Comment:
Do you think it would be a good idea to wrap `datetime.datetime.now(eastern).strftime("%Y-%m-%dT%H:%M:%S.%f")` inside a function? This way the func can be referenced in the print statements instead of duplicating code.